src/Spout/Reader/XLSX/Helper/SheetHelper.php

```diff
@@ -73,6 +73,28 @@ public function getSheets()
     }
 
     /**
+     * Get active sheet index
+     *
+     * @return int index of active sheet
+     */
+    public function getActiveSheetIndex()
+    {
+        $activeSheetIndex = 0;
+
+        $xmlReader = new XMLReader();
+        if ($xmlReader->openFileInZip($this->filePath, self::WORKBOOK_XML_FILE_PATH)) {
```

Comment:
So we basically reading the same file twice. Not good... This should be done in one pass (combining this code with the code from `getSheets`.